offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/ProjectInformationView.groovy

```diff
@@ -80,97 +74,86 @@ class ProjectInformationView extends VerticalLayout implements Resettable {
     }
 
     private void resetContent() {
-        this.projectTitle.setValue("")
-        this.projectObjective.setValue("")
-        this.experimentalDesign.setValue("")
+        this.createOfferViewModel.projectTitle = null
+        this.createOfferViewModel.projectObjective = null
+        this.createOfferViewModel.experimentalDesign = null
```

Comment:
setting these fields null creates an null pointer for me.

Reply:
df1a1ab